week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,29 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/* Add comments to explain what this function does. You're meant to use Google!*/
 function getNumber() {
     return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/*expected return: The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1 in decimal form) and then multiplying by 10. 
+
+-source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random */
+
+/* Add comments to explain what this function does. You're meant to use Google!*/
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
+/*Expected return: The concat() method concatenates the string arguments to the calling string and returns a new string.
+//source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat*/
+
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
+
+//return firstWord + " " + secondWord + " " + thirdWord;
+
+// return `${firstWord} ${secondWord} ${thirdWord}`;  - "String" ${Interpolation}
+
+return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
All of these options work! However, if you look at the [Performance section of the documentation you linked above](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance), you'll see that the string `.concat()` method isn't recommended any more. I would stick to the other options you've  given here.